lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -1884,6 +1906,46 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     visitPart(RST->getReferentType(), optionalKind);
   }
   
+  Optional<ValueDecl *> findRenamedDecl(const swift::AvailableAttr *AvAttr, const swift::Decl *D) {
```

Comment:
Nitpick: We have an 80-character column limit, but you can also drop the `swift::` namespace whenever you're in a Swift source file.